customer-terminate-service/app/src/main/java/se/inera/intyg/cts/application/service/TerminationServiceImpl.java

```diff
@@ -79,20 +75,17 @@ public List<TerminationDTO> findAll() {
 
   /**
    * Resend password if all criteria are fulfilled
```

Comment:
Was is the "all criteria" referring to now that the logic has moved to the domain service? Maybe could be removed from the javadoc?